packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAccountCard/TransferFormAccountCard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import classNames from 'classnames';
+import {
+    WalletCurrencyCard,
+    WalletListCardBadge,
+    WalletsAppLinkedWithWalletIcon,
+    WalletText,
+} from '../../../../../../components';
+import useDevice from '../../../../../../hooks/useDevice';
+import { useWalletTransfer } from '../../hooks';
+import './TransferFormAccountCard.scss';
+
+type TProps = {
+    account?: ReturnType<typeof useWalletTransfer>['activeWallet'];
+    type?: 'input' | 'modal';
+};
+
+const WalletTransferFormAccountCard: React.FC<TProps> = ({ account, type = 'modal' }) => {
+    const { isMobile } = useDevice();
+    const { activeWallet } = useWalletTransfer();
+    const isInput = type === 'input';
+    const isModal = type === 'modal';
+
+    return (
+        <div
+            className={classNames('wallets-transfer-form-account-card', {
+                'wallets-transfer-form-account-card--is-input': isInput,
+                'wallets-transfer-form-account-card--is-modal': isModal,
+            })}
+        >
+            <div className='wallets-transfer-form-account-card__icon-with-badge'>
+                <div
+                    className={classNames('wallets-transfer-form-account-card__icon', {
+                        'wallets-transfer-form-account-card__icon--is-input': isInput,
+                        'wallets-transfer-form-account-card__icon--is-modal': isModal,
```

Comment:
there are duplicated modifiers (--is-input, --is-modal) on both top-level div and within form-account__icon, 

it should be enough to avoid duplication and keep those modifiers on top level of component (generally its good habit to keep modifiers only on top level of structure of file whenever it makes sense)